docs/configuration/configuration.md

```diff
@@ -1349,15 +1349,15 @@ interface.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_openstack_address_pool`: the pool of the private IP.
-* `__meta_openstack_instance_flavor`: the flavor of the OpenStack instance.
+* `__meta_openstack_instance_flavor`: the flavor ID of the OpenStack instance.
 * `__meta_openstack_instance_id`: the OpenStack instance ID.
 * `__meta_openstack_instance_image`: the ID of the image the OpenStack instance is using.
 * `__meta_openstack_instance_name`: the OpenStack instance name.
 * `__meta_openstack_instance_status`: the status of the OpenStack instance.
 * `__meta_openstack_private_ip`: the private IP of the OpenStack instance.
 * `__meta_openstack_project_id`: the project (tenant) owning this instance.
 * `__meta_openstack_public_ip`: the public IP of the OpenStack instance.
-* `__meta_openstack_tag_<tagkey>`: each tag value of the instance.
+* `__meta_openstack_tag_<key>`: each metadata item of the instance, with any unsupported characters converted to an underscore.
```

Comment:
```suggestion
* `__meta_openstack_tag_<key>`: each metadata item of the instance, with any unsupported character converted to an underscore.
```
(Given how confusing this may be, we can even add that it has nothing to do with [the instance tags](https://docs.openstack.org/api-ref/compute/#server-tags-servers-tags))

Reply:
I believe that we should keep "characters" (I copied the text from other SD descriptions).

Reply:
I would expect two successive unsupported characters to be converted to 2 underscores instead of one and I think it's what we do https://github.com/prometheus/prometheus/blob/8894d65cd6135635c4ac9cf100464e6aacd29593/util/strutil/strconv.go#L40-L47
Ok, as it doesn't seem to be confusing elsewhere and to keep consistency, yes let's merge as it is.